stackit/internal/services/ske/project/resource.go

```diff
@@ -86,7 +86,7 @@ func (r *projectResource) Configure(ctx context.Context, req resource.ConfigureR
 // Schema returns the Terraform schema structure
 func (r *projectResource) Schema(_ context.Context, _ resource.SchemaRequest, resp *resource.SchemaResponse) {
 	resp.Schema = schema.Schema{
-		Description: "SKE project resource schema. Must have a `region` specified in the provider configuration.",
+		Description: "SKE project resource schema. Must have a `region` specified in the provider configuration. Deleting this resource will destroy any SKE clusters associated to the project",
```

Comment:
Probably it's also worth documenting that only one `stackit_ske_project` resource should be created, and that this only enables the SKE functionality for the given STACKIT project